### PR TITLE
Fix speed of light

### PR DESCRIPTION
**Pull request by jacg**
The speed of light was out by a factor of 10^9. I'm still puzzled by why this does not seem to have had grave consequences for reconstruction.

### CODE REVIEW

**Comment by jacg** [comment]
Fixing the speed of light (factor of 10^9) seems to give a slight improvement in FOMs:
```
tail -1 deleteme-before-fixed-c/foms
04-20   53.7   60.3   66.9   73.1   70.0   75.5            26.1   24.7   24.4   24.4   24.0   23.4           304.0  325.2  226.3  331.0  249.8  270.2 

tail -1 deleteme-fixed-c/foms
04-20   56.1   61.5   68.0   74.5   71.4   77.4            21.7   20.9   19.9   19.9   19.8   19.6           378.5  358.4  256.7  365.5  272.6  298.5 
```
(The groups are CRCs, background variability and SNR.)

This used an attenuation image which was made before the fix, but off the top of my head I can't think of a reason why that would depend on c.

**Comment by jacg** [comment]
This fixes a problem in `vislor` where the TOF option appeared not to work, because whatever TOF you specified was interpreted as 10^9 times that value, leading to a gaussian which was essentially constant across the whole FOV.

File: geometry/src/units.rs [hunk 1]
```diff
@@ -94,6 +94,10 @@ pub fn ratio_ (x: Ratio) -> f32 { x.get::<uom::si::ratio::ratio>() }
 pub fn radian_(x: Angle) -> f32 { x.get::<uom::si::angle::radian>() }
 pub fn turn_  (x: Angle) -> f32 { x.get::<uom::si::angle::revolution>() }
 
+pub fn mm_ps (x: f32) -> Velocity { m_s (x * 1e+9) }
+pub fn mm_ps_(x: Velocity) -> f32 { m_s_(x * 1e-9) }
```
Comment (andLaing) [inline geometry/src/units.rs:98]: I'm not a huge fan of the hard wired `1e9`. Maybe it'd be more robust to have a function or constant that does the conversion. Without too much thinking I found:

`fn mpsec() -> f32 {
  Velocity::new::<meter_per_second>(1.0).value
}
pub fn mm_ps (x: f32) -> Velocity { m_s (x / mpsec()) }
pub fn mm_ps_(x: Velocity) -> f32 { m_s_(x * mpsec()) }`

But there's maybe something a bit less brute force.

**Review by andLaing — APPROVED** [review]
I think it's acceptable as is but maybe it could be improved by not using the hard wired conversion factor.

File: geometry/src/units.rs (hunk 1 above)
Comment (jacg) [reply geometry/src/units.rs:98]: Oh, I do like this.

(I've edited your comment: it was very difficult to read with single backticks.)

**Review by andLaing — APPROVED** [review]
Simple change to fix a bug, tests added to protect.

**Pull request merged**